sdcm/utils/common.py

```diff
@@ -951,3 +951,21 @@ def makedirs(path):
         if os.path.exists(path):
             return
         raise
+
+
+def is_ami_available(client, ami_id):
```

Comment:
If the AMI is preparing, and not available, this function will stuck and return when it's available or timeout (raise exception)?

The function returns nothing, it seems not a check function.  how about renaming it to  wait_image_available()?

Reply:
renamed.
yes function will wait for ami readyness for 10 minutes. each 30 seconds will be send request 20 times. if it will not be ready, raise exception, ami is not ready(boto3 api)